src/directives/initializer.ts

```diff
@@ -51,6 +52,8 @@ export function DirectiveWrapper(flag: Flag): Directive | undefined {
 					return new DirectiveHealPoint(flag);
 				case COLOR_CYAN:
 					return new DirectiveDestroy(flag);
+				case COLOR_YELLOW:
```

Comment:
Can you change to red/purple? Red is any offensive directives (blue will be defensive in future) and purple is generally things having to do with controllers (I guess the thought process there is purple = royalty/empire = territory)